packages/block-editor/src/date-time-picker/edit.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+
+import { __ } from '@wordpress/i18n';
+import classnames from 'classnames';
+import { RichText } from '@wordpress/block-editor';
+import DatePicker from 'react-datepicker';
+
+/**
+ * Internal dependencies
+ */
+import { FormInputWrapper } from '@crowdsignal/blocks';
+import { useColorStyles } from '@crowdsignal/styles';
+import Sidebar from './sidebar';
+import { useClientId } from '@crowdsignal/hooks';
+
+const EditDateTimePicker = ( props ) => {
+	const { attributes, setAttributes, className } = props;
+
+	useClientId( props );
+
+	const handleChangeLabel = ( label ) => setAttributes( { label } );
+	const classes = classnames(
```

Comment:
Small nitpick, but all the others have a space between the lines, probably should stay consistent.

Reply:
✔️ Thanks!  good catch